phaseshifts/factories.py

```
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# encoding: utf-8
```

Ruff (0.14.10)

UTF-8 encoding declaration is unnecessary

Remove unnecessary coding comment

(UP009)